.travis/code-climate.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
I have noticed that there are files related to setting up / configuring things on Travis CI in [`/tools/travis`](https://github.com/opencfp/opencfp/tree/master/tools/travis) - shall we move this file there, or maybe move the other files here?